docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -1,283 +1,258 @@
+If you are launching your Metabase inside of a VPC you'll now need to check a few boxes to enable your application to work inside your VPC subnets. Click `Modify` on this Network box to set things up.
 
 ![Elastic Beanstalk VPC Entry](images/EBVPCEntry.png)
 
-If you are launching your Metabase inside of a VPC you'll now need to check a few boxes to enable your application to work inside your VPC subnets.
-
-Unless you have a custom VPC setup that you know how to configure it's easiest to just check all the boxes and allow your infrastructure to exist on all subnets. Note that the Load Balancer cannot be in more than one subnet per availability zone, and the database requires subnets in at least two availability zones.
+Unless you have a custom VPC setup that you know how to configure, it's easiest to just check all the boxes and allow your infrastructure to exist on all subnets. Note that the Load Balancer cannot be in more than one subnet per availability zone, and the database requires subnets in at least two availability zones. Once you've finished your VPC config click `Save`.
 
 ![Elastic Beanstalk VPC Settings](images/EBVPCSettings.png)
 
-Once you've finished your VPC config click `Save`
-
 ### Configuration Details
 
 There are many ways to customize your Beanstalk deployment, but commonly modified settings include:
 
-* `Instance type` is for picking the size of AWS instance you want to run.  Any size is fine but we recommend `t2.small` for most uses.
-    * Remember that you cannot choose a t2.* instance type if you did not check the box to run in a VPC.
-* `EC2 key pair` is only needed if you want to ssh into your instance directly.  We recommend leaving this out.
-* Enter an `Email address` to get notifications about your deployments and changes to your application.  This is a very simple way to keep tabs on your Metabase environment, so we recommend putting a valid email in here.
-* The `Application health check URL` is how Elastic Beanstalk knows when the application is ready to run. You must set this to `/api/health`
-* The remainder of the options can all be safely left to their default values
-
+- `Instance type` (`Instances` block) is for picking the size of AWS instance you want to run. Any size is fine but we recommend `t2.small` for most uses.
+  - Remember that you cannot choose a `t2.*` instance type if you did not check the box to run in a VPC.
+- `EC2 key pair` (`Security` block) is only needed if you want to SSH into your instance directly. We recommend leaving this out.
+- Enter an `Email address` (`Notifications` block) to get notifications about your deployments and changes to your application. This is a very simple way to keep tabs on your Metabase environment, so we recommend putting a valid email in here.
+- The `Application health check URL` is how Elastic Beanstalk knows when the application is ready to run. You must set this to `/api/health`.
```

Comment:
I _think_ this is updated from the "Load balancer" config, under "Processes" :man_shrugging:: 
![image](https://user-images.githubusercontent.com/23798/66218023-a48c0c00-e6c8-11e9-99e9-05f86f67dc23.png).

As you can see, this is **not** set to `/api/health` by default, and yet everything works after deploying the app.

Reply:
Hrm. I think I'm going to leave this for now until we're more certain it doesn't need to be set.